notebooks/data-sources/TestGrid/metrics/build_pass_failure.ipynb

```diff
@@ -2,25 +2,25 @@
  "cells": [
```

Comment:
<ul><li>"Total number of tests:" should this be total number of "builds"?</li></ul><p> </p><ul><li>failing build % = 42.5</li></ul><p>			passing build % = 41.1</p><p>			What are the other 16% of builds doing? Does it have something to do with managing two DF's <code>build_pass</code>    and 		<code>build_fail</code>  . Can't all these values get calculated based only on the build passing df? Shouldn't these two df's be identical except for opposite values in their boolean column at the end?</p><p> </p><ul><li>no_tests = build_passing_df.head(1000000).test.count() = 32,734 so, limiting the DF to the first million rows is not actually doing any work here, you should remove it throughout. </li></ul><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/172/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>The label value can take one of 16 values in https://github.com/aicoe-aiops/ocp-ci-analysis/blob/1a3a7a58a340c401f8c68ef5d3e9c3f8e11ef864/docs/cell_labels_in_testgrid.md</p><p> </p><p>Although Pass and Fail labels are more common, there could be occurences of Running for the Overall Label, therefore it probably makes sense to calculate "Passing" and "Failing" labels separately.</p><p> </p><p>yes, can remove the (100000) limit everywhere :thumbs_up:</p>

Reply:
<p>Made the requested changes</p>